code/game/objects/structures/crates_lockers/closets/secure/supplypod.dm

```diff
@@ -104,7 +104,7 @@
 	light_range = 2
 	var/obj/effect/temp_visual/fallingPod
 
-/obj/effect/DPtarget/Initialize(mapload, var/SO, var/podID, var/target)
+/obj/effect/DPtarget/Initialize(mapload, var/SO, var/podID)

```

Comment:
Remove `var/`s

Reply:
yikes i should do that from everything huh